.github/workflows/tests.yml

```
@@ -3,6 +3,10 @@
 
 name: Tests
 
+concurrency:
+  group: ${{ github.event.pull_request.id || github.ref }}
```

Sorry for nits after another :D Let's specify this group name, so it doesn't coincide with other concurrency groups we might add later.

```suggestion
  group: e2e-${{ github.event.pull_request.id || github.ref }}
```